packages/api/endpoints/granules.js

```diff
@@ -549,14 +549,15 @@ async function bulkOperations(req, res) {
       type: 'BULK_GRANULE',
       envVars: {
         ES_HOST: process.env.ES_HOST,
-        GranulesTable: process.env.GranulesTable,
         granule_sns_topic_arn: process.env.granule_sns_topic_arn,
-        system_bucket: process.env.system_bucket,
-        stackName: process.env.stackName,
+        GranulesTable: process.env.GranulesTable,
         invoke: process.env.invoke,
+        KNEX_DEBUG: payload.knexDebug || false,
```

Comment:
Should we explicitly check this? It's going to be a string I guess so if I submit a payload with `KNEX_DEBUG: 'False'` this would turn out truthy.

Maybe `toLowerCase(payload.knexDebug) === 'true' || false` to not accept any string value.

Reply:
Sure thing: 8e3b3f9635

Reply:
8e3b3f9635